src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages.properties

```diff
@@ -30,6 +30,11 @@ ascanalpha.httpsashttpscanner.desc = Content which was initially accessed via HT
 ascanalpha.httpsashttpscanner.soln = Ensure that your web server, application server, load balancer, etc. is configured to only serve such content via HTTPS. Consider implementing HTTP Strict Transport Security.
 ascanalpha.httpsashttpscanner.refs = https://www.owasp.org/index.php/HTTP_Strict_Transport_Security\nhttps://www.owasp.org/index.php/List_of_useful_HTTP_headers\nhttp://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security\nhttp://caniuse.com/stricttransportsecurity\nhttp://tools.ietf.org/html/rfc6797
 
+ascanalpha.httponlysite.name = HTTP Only Site
+ascanalpha.httponlysite.desc = The site is only served under HTTP and not HTTPS
+ascanalpha.httponlysite.soln = None
```

Comment:
Configure you web or application server to use SSL (https).

Or something like that.